common/model-views.cpp

```diff
@@ -3379,9 +3379,14 @@ namespace rs2
         , _detected_objects(std::make_shared< atomic_objects_in_frame >()),
         _updates(viewer.updates)
     {
-        if (dev.is<device_calibration>())
+        if( dev.supports( RS2_CAMERA_INFO_PRODUCT_LINE )
```

Comment:
Why? Something was wrong with the device_calibration?

Reply:
It's still there, view the entire file. the order is a bit changed and formatted differently, not shown here.